Tests/Brokerages/Binance/BinanceBrokerageAdditionalTests.cs

```diff
@@ -0,0 +1,132 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using Moq;
+using NUnit.Framework;
+using QuantConnect.Brokerages;
+using QuantConnect.Brokerages.Binance;
+using QuantConnect.Configuration;
+using QuantConnect.Interfaces;
+using QuantConnect.Lean.Engine.DataFeeds;
+using QuantConnect.Securities;
+using QuantConnect.Tests.Common.Securities;
+using System;
+using QuantConnect.Util;
+
+namespace QuantConnect.Tests.Brokerages.Binance
+{
+    [TestFixture, Explicit("This test requires a configured and testable Binance practice account")]
+    public class BinanceBrokerageAdditionalTests
+    {
+        [Test]
+        public void ParameterlessConstructorComposerUsage()
+        {
+            var brokerage = Composer.Instance.GetExportedValueByTypeName<IDataQueueHandler>("BinanceBrokerage");
+            Assert.IsNotNull(brokerage);
+            Assert.True(brokerage.IsConnected);
+        }
+
+        [Test]
+        public void ConnectedIfNoAlgorithm()
+        {
+            var securities = new SecurityManager(new TimeKeeper(DateTime.UtcNow, TimeZones.NewYork));
+
+            var transactions = new SecurityTransactionManager(null, securities);
+            transactions.SetOrderProcessor(new FakeOrderProcessor());
+
+            var apiKey = Config.Get("binance-api-key");
+            var apiSecret = Config.Get("binance-api-secret");
+            var apiUrl = Config.Get("binance-api-url", "https://api.binance.com");
+            var websocketUrl = Config.Get("binance-websocket-url", "wss://stream.binance.com:9443/ws");
+
+            using var brokerage = new BinanceBrokerage(
+                apiKey,
+                apiSecret,
+                apiUrl,
+                websocketUrl,
+                null,
+                new AggregationManager(),
+                null
+            );
+
+            Assert.True(brokerage.IsConnected);
+        }
+
+        [Test]
+        public void ConnectedIfAlgorithmIsNotInitialized()
+        {
+            var securities = new SecurityManager(new TimeKeeper(DateTime.UtcNow, TimeZones.NewYork));
+
+            var transactions = new SecurityTransactionManager(null, securities);
+            transactions.SetOrderProcessor(new FakeOrderProcessor());
+
+            var apiKey = Config.Get("binance-api-key");
+            var apiSecret = Config.Get("binance-api-secret");
+            var apiUrl = Config.Get("binance-api-url", "https://api.binance.com");
+            var websocketUrl = Config.Get("binance-websocket-url", "wss://stream.binance.com:9443/ws");
+
+            using var brokerage = new BinanceBrokerage(
+                apiKey,
+                apiSecret,
+                apiUrl,
+                websocketUrl,
+                null,
```

Comment:
- `null` no algorithm is being provided here really?
- There's some duplication in theses tests which we could push into a private method/test setup